babel/src/main/codegen/config.fmpp

```diff
@@ -141,6 +141,7 @@ data: {
         "JSON"
         "JSON_TYPE"
         "JSON_DEPTH"
+        "JSON_LENGTH"
```

Comment:
Some JSON-related keywords seem to appear twice in the list. Could you please remove the redundant ones?